Makefile

```diff
@@ -1,7 +1,11 @@
 VERSION := $(shell cat VERSION)
+PY2VERSION := $(shell cat PY2VERSION)
 
-.PHONY: build
-build:
+.PHONY: build-all
+build-all: build-py3 build-py2
```

Comment:
I'd keep these as `build` and `push`. Same idea as the version file - once we stop using PY2 images we'll have only one task for each.